src/event/event-publisher.service.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { DaprClient, CommunicationProtocolEnum } from '@dapr/dapr';
+
+/**
+ * Service for publishing events to Dapr pubsub components.
+ */
+@Injectable()
+export class EventPublisherService {
+  private readonly daprClient: DaprClient;
```

Comment:
ideally these two would have docs

Reply:
Changed in [c5b002f89a4dc648aa45f4d818d4471103ea8c87](https://github.com/MiSArch/experiment-config/pull/1/commits/c5b002f89a4dc648aa45f4d818d4471103ea8c87)